projects/Vula/test.nix

```diff
@@ -0,0 +1,42 @@
+{
+  sources,
+  pkgs,
+  ...
+}: {
+  name = "vula";
+
+  nodes = {
+    server = {config, ...}: {
+      imports = [
+        sources.modules.default
+        sources.modules."services.vula"
+      ];
+      services.vula.enable = true;
+    };
+  };
+
+  testScript = {nodes, ...}: ''
+    start_all()
```

Comment:
@wegank this is not a test. Did you test Vula in any way manually?

Reply:
Oh, no. Should I drop test.nix in another PR?

Reply:
No worries, that'll be on my head, since I left this PR open. The issue I ran into with this PR is mentioned in https://github.com/ngi-nix/ngipkgs/issues/96#issuecomment-1867639853

I'd say we keep `test.nix` as-is for now, but leave #96 open until we have a fix.

Ideally, it would be great if you could diagnose and fix this, but I am not sure about your capacity.

Reply:
@ngi-nix/mob-cake xor @ngi-nix/mob-magic could take on the debugging.

Reply:
Anyone: In case you start working on this, please comment here and in #96 and resolve this conversation. Thanks.